CMakeLists.txt

```diff
@@ -623,6 +623,7 @@ if(BUILD_TESTS)
     NAME e2e_logging_cft
     PYTHON_SCRIPT ${CMAKE_SOURCE_DIR}/tests/e2e_logging.py
     CONSENSUS cft
+    DOCKER_NODES TRUE
```

Comment:
Can we run this in Daily only? I imagine it's going to be slower, and possibly less stable than the current setup at least at first.